README.md

```diff
@@ -1,6 +1,6 @@
 # Sofie: The Modern TV News Studio Automation System (tv-automation-quantel-browser-plugin)
 
-A MOS-compatible Quantel video clip browser for [**Sofie** TV News Studio Automation System](https://github.com/nrkno/Sofie-TV-automation/).
+A MOS-compatible Quantel and CasparCG Scanner video clip browser for [**Sofie** TV News Studio Automation System](https://github.com/nrkno/Sofie-TV-automation/).
```

Comment:
Should we rename this repository/project to clarify this? `sofie-clip-browser-mos-plugin`?